regions/core/mask.py

```diff
@@ -110,35 +135,42 @@ def to_image(self, shape):
         if len(shape) != 2:
             raise ValueError('input shape must have 2 elements.')
 
-        mask = np.zeros(shape)
+        image = np.zeros(shape)
+
+        if self.bbox.ixmin < 0 or self.bbox.iymin < 0:
+            return self._to_image_partial_overlap(image)
 
         try:
-            mask[self.bbox.slices] = self.data
+            image[self.bbox.slices] = self.data
         except ValueError:    # partial or no overlap
-            slices_large, slices_small = self._overlap_slices(shape)
+            image = self._to_image_partial_overlap(image)
 
-            if slices_small is None:
-                return None    # no overlap
-
-            mask = np.zeros(shape)
-            mask[slices_large] = self.data[slices_small]
-
-        return mask
+        return image
 
-    def cutout(self, data, fill_value=0.):
+    def cutout(self, data, fill_value=0., copy=False):
         """
         Create a cutout from the input data over the mask bounding box,
         taking any edge effects into account.
 
         Parameters
         ----------
-        data : array_like or `~astropy.units.Quantity`
+        data : array_like
             A 2D array on which to apply the region mask.
 
         fill_value : float, optional
             The value is used to fill pixels where the region mask
             does not overlap with the input ``data``.  The default is 0.
 
+        copy : bool, optional
+            If `True` then the returned cutout array will always be hold
+            a copy of the input ``data``.  If `False` and the mask is
+            fully within the input ``data``, then the returned cutout
+            array will be a view into the input ``data``.  In cases
+            where the mask partially overlaps or has no overlap with the
+            input ``data``, the returned cutout array will always hold a
+            copy of the input ``data`` (i.e. this keyword has no
+            effect).
+
         Returns
         -------
         result : `~numpy.ndarray`
```

Comment:
```result : `~numpy.ndarray` or None``` Just a suggestion.

Reply:
Yes, thanks.  I also updated this in the `multiply` method.